### PR TITLE
[MTM-46157]-Add-requirements-for-managing-user-hierarchies

### PR DESCRIPTION
**Pull request by Airdroplet**
* added requirements for user hierarchy feature enablement

### CODE REVIEW

File: content/users-guide/enterprise-tenant-bundle/user-hierarchies.md [hunk 1]
```diff
@@ -6,11 +6,29 @@ aliases:
...
 ---
 
-With user hierarchies you can reflect independent organizational entities in {{< product-c8y-iot >}} that still share the same database. These entities can have limited permissions to subsets of the shared data and can manage their own sub-users.
+{{< c8y-admon-req >}}
+SUBSCRIPTIONS:
 
-{{< c8y-admon-info >}}
-To be able to use this feature, your tenant must be subscribed to the application "feature-user-hierachy".
-{{< /c8y-admon-info >}}
+Tenant must be subscribed to the application "feature-user-hierachy".
```
Comment (BeateRixen) [inline content/users-guide/enterprise-tenant-bundle/user-hierarchies.md:12]:
```suggestion
The tenant must be subscribed to the application "feature-user-hierachy".
```

File: content/users-guide/enterprise-tenant-bundle/user-hierarchies.md [hunk 2]
```diff
@@ -6,11 +6,29 @@ aliases:
...
+
+APPLICATION ACCESS:
+
+To access the feature, application access should be granted for user for "Administration" application.
```
Comment (BeateRixen) [inline content/users-guide/enterprise-tenant-bundle/user-hierarchies.md:16]:
```suggestion
To access the feature, application access must be granted to the user for the Administration application.
```

File: content/users-guide/enterprise-tenant-bundle/user-hierarchies.md [hunk 3]
```diff
@@ -6,11 +6,29 @@ aliases:
...
+
+ROLES & PERMISSIONS:
+
+"User management" permission type:
+* To view all users: READ permission.
+* To manage all users: ADMIN permission.
+* To create users: CREATE permission. By assigning this permission you can create own subusers and manage underlying hierarchy of users.
```
Comment (BeateRixen) [inline content/users-guide/enterprise-tenant-bundle/user-hierarchies.md:23]:
```suggestion
* To create users: CREATE permission. By assigning this permission you can create own sub-users and manage the underlying hierarchy of users.
```

File: content/users-guide/enterprise-tenant-bundle/user-hierarchies.md [hunk 4]
```diff
@@ -6,11 +6,29 @@ aliases:
...
+
+On tenant creation, there are default roles available that can be used as a sample configuration for the above-mentioned permissions:
```
Comment (BeateRixen) [inline content/users-guide/enterprise-tenant-bundle/user-hierarchies.md:25]:
```suggestion
On tenant creation, default roles are available which can be used as a sample configuration for the above-mentioned permissions:
```

File: content/users-guide/enterprise-tenant-bundle/user-hierarchies.md [hunk 5]
```diff
@@ -6,11 +6,29 @@ aliases:
...
+* Global User Manager - Can access and modify the full user hierarchy (all users).
+* Shared User Manager - Can create new users as his own subusers and manage underlying users' hierarchy.
```
Comment (BeateRixen) [inline content/users-guide/enterprise-tenant-bundle/user-hierarchies.md:27]:
```suggestion
* Shared User Manager - Can create new users as his own sub-users and manage the underlying user hierarchy.
```

File: content/users-guide/enterprise-tenant-bundle/user-hierarchies.md [hunk 6]
```diff
@@ -6,11 +6,29 @@ aliases:
...
+* To manage all users: ADMIN permission.
+* To create users: CREATE permission. By assigning this permission you can create own sub-users and manage the underlying hierarchy of users.
+
+On tenant creation, default roles are available which can be used as a sample configuration for the above-mentioned permissions:
+* Global User Manager - Can access and modify the full user hierarchy (all users).
+* Shared User Manager - Can create new users as his own sub-users and manage the underlying user hierarchy.
+
+{{< /c8y-admon-req >}}
+
+With user hierarchies you can reflect independent organizational entities in {{< product-c8y-iot >}} that still share the same database. These entities can have limited permissions to subsets of the shared data and can manage their own sub-users.
```
Comment (pawel-rynarzewski-sag) [inline content/users-guide/enterprise-tenant-bundle/user-hierarchies.md:31]: "t independent organizational entities in Cumulocity IoT" sounds a bit weird, maybe:
```suggestion
With user hierarchies you can model the structure of organizational entities in your company which share the same database in {{< product-c8y-iot >}} platform. These entities can have limited permissions to subsets of the shared data and can manage their own sub-users.
```

File: content/users-guide/enterprise-tenant-bundle/user-hierarchies.md [hunk 7]
```diff
@@ -6,11 +6,29 @@ aliases:
...
+The tenant must be subscribed to the application "feature-user-hierachy".
+
+APPLICATION ACCESS:
+
+To access the feature, application access must be granted to the user for the Administration application.
+
+ROLES & PERMISSIONS:
+
+"User management" permission type:
+* To view all users: READ permission.
```
[the remark this answers is not in the document]
Comment (Airdroplet) [reply content/users-guide/enterprise-tenant-bundle/user-hierarchies.md:21]: There is some guides we used before, lets keep original approach for now, ref:
content/users-guide/cockpit-bundle/exports.md
content/users-guide/administration-bundle/managing-users.md

File: content/users-guide/enterprise-tenant-bundle/user-hierarchies.md [hunk 8]
```diff
@@ -6,11 +6,29 @@ aliases:
...
+* To manage all users: ADMIN permission.
```
[the remark this answers is not in the document]
Comment (Airdroplet) [reply content/users-guide/enterprise-tenant-bundle/user-hierarchies.md:22]: There is some guides we used before, lets keep original approach for now, ref:
content/users-guide/cockpit-bundle/exports.md
content/users-guide/administration-bundle/managing-users.md

File: content/users-guide/enterprise-tenant-bundle/user-hierarchies.md [hunk 9]
```diff
@@ -6,11 +6,29 @@ aliases:
...
+* To create users: CREATE permission. By assigning this permission you can create own sub-users and manage the underlying hierarchy of users.
```
[the remark this answers is not in the document]
Comment (Airdroplet) [reply content/users-guide/enterprise-tenant-bundle/user-hierarchies.md:23]: There is some guides we used before, lets keep original approach for now, ref:
content/users-guide/cockpit-bundle/exports.md
content/users-guide/administration-bundle/managing-users.md

File: content/users-guide/enterprise-tenant-bundle/user-hierarchies.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (BeateRixen) [reply content/users-guide/enterprise-tenant-bundle/user-hierarchies.md:21]: I agree. Let´s keep it consistent. Moreover, I think it is clearer from a user perspective to first state what you want to do and second which permission you need to do so. Here it is quite simple but we have much more complex permissions.

**Review by BeateRixen — APPROVED** [review]

**Review by pawel-rynarzewski-sag — APPROVED** [review]

**Pull request merged**